src/decoder/plugins/SidplayDecoderPlugin.cxx

```diff
@@ -65,6 +66,38 @@ static unsigned default_songlength;
 
 static bool filter_setting;
 
+#ifdef HAVE_SIDPLAYFP
+static const unsigned romSize = 8192;
+static std::unique_ptr<uint8_t[]> kernal(new uint8_t[romSize]());
+static std::unique_ptr<uint8_t[]> basic(new uint8_t[romSize]());
+
+static void loadRom(std::unique_ptr<uint8_t[]> &dump, const Path romPath)
```

Comment:
There's no point in passing the `std::unique_ptr` here, because that pointer will not be modified by the function; just pass `uint8_t *`; and I'd prefer to have the `Path` parameter first. Also, don't use camelCase for variable names; underscore instead.

Reply:
Convention. No technical reason.